### PR TITLE
Update create account page content

### PR DESCRIPTION
**Pull request by bethwalton**
- Updated eligibility criteria for Gateway and Verify 
- Added examples of the most popular other services you can use each account for 
- Edited headings and buttons to make it clearer that users need to choose between two options

This is part of work on GOV.UK Verify user journeys, being done by the hub/eIDAS team. The new page content has been approved by HMRC.

[Jira ticket is here. ](https://govukverify.atlassian.net/browse/HUB-660?atlOrigin=eyJpIjoiMTkxNzU0MzY5YTc1NDc0Yzg5ZGE4NDQ3ZjlkMjAxMjUiLCJwIjoiaiJ9)

### CODE REVIEW

File: lib/service_sign_in/check-income-tax.en.yaml [hunk 1]
```diff
@@ -17,37 +17,35 @@ choose_sign_in:
...
 
-    Registering with Government Gateway usually takes about 10 minutes. It works best if you have:
+    - your National Insurance number or UK address
+    - a recent payslip or P60 or a valid UK passport
+    
+    You can use your user ID for things like your personal or business tax account, Self Assessment, Corporation Tax, PAYE for employers and VAT.
 
-    * your National Insurance number
-    * a recent payslip or P60 or a valid UK passport
+    {button}[Choose Government Gateway](https://www.tax.service.gov.uk/gg/sign-in?continue=/check-income-tax/what-do-you-want-to-do&accountType=individual&origin=PTA-CYIT-CY){/button}
```
Comment (ChrisBAshton) [inline lib/service_sign_in/check-income-tax.en.yaml:36]: I know you just moved this code a little further up, but I suspect the `origin` is wrong here - should it be `PTA-CYIT-EN`?

File: lib/service_sign_in/check-state-pension.en.yaml [hunk 2]
```diff
@@ -20,34 +20,35 @@ choose_sign_in:
...
 
-    Registering with Government Gateway usually takes about 10 minutes. It works best if you have:
+    - your National Insurance number or UK address
+    - a recent payslip or P60 or a valid UK passport
+    
+    You can use your user ID for things like your personal or business tax account, Self Assessment, Corporation Tax, PAYE for employers and VAT.
 
-    * your National Insurance number
-    * a recent payslip or P60 or a valid UK passport
+    <a role="button" class="gem-c-button govuk-button" href="https://www.tax.service.gov.uk/gg/sign-in?continue=/check-your-state-pension&accountType=individual" data-module="cross-domain-tracking" data-tracking-code="UA-43414424-1" data-tracking-name="govspeakButtonTracker" data-tracking-track-event="true">Choose Government Gateway</a>
```
Comment (ChrisBAshton) [inline lib/service_sign_in/check-state-pension.en.yaml:39]: I wonder why this file uses raw HTML but the previous one uses GovSpeak? 🤔

Comment (bethwalton) [reply lib/service_sign_in/check-state-pension.en.yaml:39]: Yeah I thought that was strange... I've never updated these pages before and I didn't want to mess with anything I didn't need to though, so I left it alone!

File: lib/service_sign_in/check-income-tax.en.yaml (hunk 1 above)
Comment (bethwalton) [reply lib/service_sign_in/check-income-tax.en.yaml:36]: CY as in the Welsh version? I don't know enough to say for sure I'm afraid. 

I just looked at the Welsh version expecting that to have the same link, but it looks like it's the wrong one altogether - it mentions 'company car' which is a different service... 

I can go back to HMRC on this.

Comment (ChrisBAshton) [reply lib/service_sign_in/check-income-tax.en.yaml:36]: No problem. Let's merge this for now, as this is the tracking link currently used on production anyway.

**Pull request merged**